app/views/episode_media/_form_status.html.erb

```diff
@@ -21,7 +21,7 @@
         <%= form.submit t(".update"), class: "btn btn-primary", data: data %>
       <% end %>
     </div>
-    <p class="status-text mt-2 mb-0 text-end"><strong><%= t(".updated_at_hint") %>:</strong> <%= l(episode.updated_at) %></p>
+    <p class="status-text mt-2 mb-0 text-end"><strong><%= t(".updated_at_hint") %>:</strong> <%= local_time_ago(form.object.updated_at) %></p>
```

Comment:
`local_time_ago` is the labeling method for episodes and podcasts so i've changed it here and below. and `episode` has been switched out for `form.object` to represent `media_resource` on this line and `transcript` below.